gammapy/modeling/models/cube.py

```diff
@@ -1202,7 +1202,7 @@ def unfreeze(self, model_type="spectral"):
             self._spectral_model.unfreeze()
 
 
-def create_fermi_isotropic_diffuse_model(filename, **kwargs):
+def create_fermi_isotropic_diffuse_model(filename, dataset_name=None, **kwargs):
```

Comment:
I dont know if its better to have `datasets_names` here, because thats what most models take as argument, and there might be cases (eg: fitting an EBL model to multiple blazar observations simultaneously) where there are multiple fermi datasets

Reply:
And if there are multiple psf classes being fit simulatneously?

Reply:
@QRemy But I guess there is the use case of e.g.  a joint HESS / Fermi analysis with PSF classes. How is this handled correctly? In this case you only want the iso model to go to the multiple  Fermi datasets and not to the HESS ones.